gamemodes/terrortown/gamemode/shared/hud_elements/tttminiscoreboard/pure_skin_miniscoreboard.lua

```diff
@@ -117,7 +117,7 @@ if CLIENT then
 		return hook.Run("TTT2ModifyMiniscoreboardColor", ply, color) or color
 	end
 
-	hook.Add("TTT2ConfirmedBody", "TTT2UpdateMiniscoreboardFound", function()
+	hook.Add("TTT2ConfirmedPlayer", "TTT2UpdateMiniscoreboardFound", function()
```

Comment:
Unless you're working with plugins/modules, you should not use hooks and directly call the function from the GM:TTT2ConfirmedPlayer

I know it's "easier" to use hooks but you should really not since you have no control in the execution order.

Reply:
You're right, that's an issue I observed on multiple occurrences in our project, but as we will rewrite many parts in the near future i will keep a look on such calls. Thanks again :+1: